.github/workflows/build-gtk.yml

```diff
@@ -38,6 +38,16 @@ jobs:
             dotnet build -c Release Mali.slnf
       - name: Pack MAUI
         run: dotnet pack Mali.slnf
+      - name: Build Mali.ZXing
+        run: |
+            cd src/ZXing.Net.Maui
+            dotnet build -c Release --framework=net8.0-gtk ZXing.Net.MAUI/ZXing.Net.MAUI.csproj
```

Comment:
@webwarrior-ws why not just add these 2 projs to our slnf?

Reply:
Because these projects are in another solution

Reply:
and you cannot build that solution with dotnet build?

Reply:
Solution `ZXing.Net.MAUI.sln` contains `BigIslandBarcode` project that we don't need to pack and that can't be build without `maui-android` workload.